samples/snippets/visualbasic/VS_Snippets_CLR/conceptual.cancellation.callback/vb/howtoexample1.vb

```diff
@@ -1,47 +1,41 @@
-﻿' Visual Basic .NET Document
-
-' <Snippet1>
+﻿Imports System.Net
 Imports System.Threading
-Imports System.Threading.Tasks
-Imports System.Net
 
-Class CancelWithCallback
-    Shared Sub Main()
-        Dim cts As New CancellationTokenSource()
-        Dim token As CancellationToken = cts.Token
+Friend Class CancelWithCallback
+    Private Shared Sub Main()
+        Using cts = New CancellationTokenSource()
+            Dim token = cts.Token
+            Task.Run(
+                Async Function()
+                    Using client As WebClient = New WebClient()
+                        AddHandler client.DownloadDataCompleted,
+                        Sub(__, args)
```

Comment:
Does VB support discards, or this is just a normal variable name?

Reply:
VB does not support discards, this is a double `__` which is common for named variables that can be ignored.